### ISSUE TITLE
Remove 'no information' section underneath page number

### ISSUE THREAD

**Issue by CypherGuy** [stshrewsburyDev/discord-better-help#5]
Hi again,

Since I have updated to 1.1, I have noticed another thing pop up.

If you look in the picture, underneath the page number, it says `no information`, circled. Is there a way to remove that bit, seeing as it doesn't really serve a purpose, or make that one line customisable from the `no_info` parameter?

Thanks again.

<img width="459" alt="Screenshot 2021-05-10 at 17 15 37" src="https://user-images.githubusercontent.com/74025064/117691121-754a5480-b1b3-11eb-91e9-c492aff7ffdc.png">

**Comment by CypherGuy** [comment]
Sorry about the title change!

**Comment by SkylaDev** [comment]
As shown in below image I added this section as it shows the description of the bot that you parse in via `commands.Bot`/`commands.AutoShardedBot`. To put info there you need to define a bot description via your client class:
```python
# Example
from discord.ext import commands
from better_help import Help

bot = commands.Bot(command_prefix="!", help_command=Help(), description="This is my bot description")
```

I did find a work around for this as I originally didnt think about bots without set descriptions.
If you dont wish to have an actual bot description for the help command you can remove the "No information" tag by setting your description to a whitespace character [Found here](https://qwerty.dev/whitespace/). I will fix this in the future and make it auto remove the "No information" tag for the bot desc but as of right now I dont have enough time to push a new update, I hope this can help for now.

![image](https://user-images.githubusercontent.com/39103023/117692397-e50d0f00-b1b4-11eb-9cb2-01b4dc7fdded.png)

**Comment by CypherGuy** [comment]
Of course, thanks for letting me know! I hope your schedule gets clearer soon.

**Comment by SkylaDev** [comment]
This problem has now been fixed in 1.2 of the module (releasing later today hopefully).
Instead of shoing the "No information" label, if there is no bot description set the description will remain empty.

**Example: (before)**
![image](https://user-images.githubusercontent.com/39103023/118354398-cca94580-b562-11eb-8ae3-b3925a2dc9da.png)

**Example: (after)**
![image](https://user-images.githubusercontent.com/39103023/118354403-d5018080-b562-11eb-85ff-ac10994113ee.png)

**Comment by CypherGuy** [comment]
Oh, that's great, thank you very much!
